Tensile/TensileCreateLibrary.py

```diff
@@ -484,30 +487,32 @@ def prepAsm(
     os.chmod(assemblerFileName, 0o777)
 
 
-################################################################################
-def buildKernelSourceAndHeaderFiles(results, outputPath):
-    """
-    Logs errors and writes appropriate info to kernelSourceFile and kernelHeaderFile.
+def buildKernelSourceAndHeaderFiles(
+    results: ProcKernResult, outputPath, lazyLoading: bool, mergeFiles: bool, numMergedFiles: int
+) -> Tuple[List[str], Dict[str, int]]:
+    """Logs errors and writes appropriate info to kernel source and header files.
```

Comment:
Logs errors for what? Are the source files and header files all cpp files?

Reply:
"Collects" is probably a better term here. Note the Return section in the doc-string.